src/frontend/components/router-tree/router-tree.ts

```diff
@@ -52,7 +52,7 @@ export class RouterTree {
     };
 
     const nodes = this.tree(d3.hierarchy(
-      root.children.length > 1 ? root : root.children[0], d => d.children));
+      root.children.length === 0 || root.children.length > 1 ? root : root.children[0], d => d.children));
```

Comment:
As further discussed I will need to revisit this short-circuit op, but look like it's working as expected!

:shipit: